cloud/src/resource-manager/resource_manager.h

```diff
@@ -114,6 +114,25 @@ class ResourceManager {
     bool check_cluster_params_valid(const ClusterPB& cluster, std::string* err,
                                     bool check_master_num);
 
+    /**
+     * Check cloud_unique_id is degraded format, and get instance_id from cloud_unique_id
+     * degraded format : "${version}:${instance_id}:${unique_id}"
+     * @param degraded cloud_unique_id
+     *
+     * @return a <is_degraded_format, instance_id> pair, if is_degraded_format == true , instance_id, if is_degraded_format == false, instance_id=""
+     */
+    std::pair<bool, std::string> get_instance_id_by_degrade_unique_id(
```

Comment:
naming "get_instance_id_by_cloud_unique_id()"
and make it a static member function